pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1760,19 +1760,33 @@ fn basic_fuzzing() {
             let instruction_counter = Settlement::instruction_counter();
 
             // initialize balances
-            for i in 0..10 {
-                for j in 0..4 {
-                    balances.insert((tickers[i * 4 + j], users[j].did, "init").encode(), 100_000);
+            for ticker_id in 0..10 {
+                for user_id in 0..4 {
                     balances.insert(
-                        (tickers[i * 4 + j], users[j].did, "final").encode(),
+                        (tickers[ticker_id * 4 + user_id], users[user_id].did, "init").encode(),
```

Comment:
The expr `tickers[ticker_id * 4 + user_id]` occurs 4x in this loop; let's dedup it for improved readability.

Reply:
Let's also dedup `users[user_id].did` while at it.